block_content_template.module

```diff
@@ -43,7 +43,14 @@ function template_preprocess_block_content_template(&$variables) {
   }
   else {
     $content = [];
-    foreach (Element::children($variables) as $key) {
+    $vars = $variables;
+    // Remove all not valid render keys.
+    foreach ($vars as $key => $var) {
```

Comment:
I'm disagree with your modifications. The drupal good practice is to use "Element::children". But the implementation in the loop is not correct.

Reply:
It's ok to disagree :). However without it invalid render keys were there and the module wasn't working, returning a fatal error. With what I did the module works and doesn't return any errors. There are many different ways of accomplishing tasks in web and it's not necessarily wrong to do things differently. It's not the foreach loop that matters in this case it's what's happening inside it. Notice how I prepare the variables and only then use Element::children. The foreach I used just cleans up the array.